apcd-cms/src/apps/exception/static/forms/css/exception_submission_form.css

```diff
@@ -0,0 +1,22 @@
+/*Text Style*/
+[id^="justification-asterisk"] {
+  padding-left: 0;
+  margin-left: 0;
+}
+/* Field Widths */
```

Comment:
Instead of a css attribute selector here could this just be selected with `#justification-asterisk`? (since this seems to be doing an exact match)

Reply:
Definitely - thank you